src/CoverageTools.jl

```diff
@@ -237,7 +237,11 @@ module CoverageTools
         coverage = process_cov(filename, folder)
         fc = FileCoverage(filename, read(filename, String), coverage)
         if get(ENV, "DISABLE_AMEND_COVERAGE_FROM_SRC", "no") != "yes"
-            amend_coverage_from_src!(fc)
+            try
+                amend_coverage_from_src!(fc)
+            catch err
+                @error "coverage could not incorporate source file $filename, it may have a parsing error" exception=(err,catch_backtrace())
+            end
```

Comment:
```suggestion
                rethrow(err)
            end
```

Reply:
It's better not to include the `err`, see
```julia
help?> rethrow
search: rethrow

  rethrow()

  Rethrow the current exception from within a catch block. The rethrown exception will continue propagation as if it had not been caught.

  │ Note
  │
  │  The alternative form rethrow(e) allows you to associate an alternative exception object e with the current backtrace. However this misrepresents the program state at the time of the error
  │  so you're encouraged to instead throw a new exception using throw(e). In Julia 1.1 and above, using throw(e) will preserve the root cause exception on the stack, as described in catch_stack.
```

Reply:
TIL!